msgraph/models.go

```diff
@@ -1250,7 +1266,8 @@ type ScopedRoleMembership struct {
 // ServicePrincipal describes a Service Principal object.
 type ServicePrincipal struct {
 	DirectoryObject
-	Owners *Owners `json:"owners@odata.bind,omitempty"`
+	Owners                *Owners `json:"owners@odata.bind,omitempty"`
+	ClaimsMappingPolicies *[]ClaimsMappingPolicy `json:"claimsmappingpolicies@odata.bind,omitempty"`
```

Comment:
Linter picked this one up, the field name, type and tag should be vertically aligned :)

```suggestion
	Owners                *Owners                `json:"owners@odata.bind,omitempty"
	ClaimsMappingPolicies *[]ClaimsMappingPolicy `json:"claimsmappingpolicies@odata.bind,omitempty"`
```

Reply:
Thank you. Fixed the formatting.